frame/staking/src/lib.rs

```diff
@@ -922,6 +960,56 @@ pub mod pallet {
 
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// The list of thresholds separating the various voter bags.
+		///
+		/// Voters are separated into unsorted bags according to their vote weight. This specifies
+		/// the thresholds separating the bags. A voter's bag is the largest bag for which the
+		/// voter's weight is less than or equal to its upper threshold.
+		///
+		/// When voters are iterated, higher bags are iterated completely before lower bags. This
+		/// means that iteration is _semi-sorted_: voters of higher weight tend to come before
+		/// voters of lower weight, but peer voters within a particular bag are sorted in insertion
+		/// order.
+		///
+		/// # Expressing the constant
+		///
+		/// This constant must be sorted in strictly increasing order. Duplicate items are not
+		/// permitted.
+		///
+		/// There is an implied upper limit of `VoteWeight::MAX`; that value does not need to be
+		/// specified within the bag. For any two threshold lists, if one ends with
+		/// `VoteWeight::MAX`, the other one does not, and they are otherwise equal, the two lists
+		/// will behave identically.
+		///
+		/// # Calculation
+		///
+		/// It is recommended to generate the set of thresholds in a geometric series, such that
+		/// there exists some constant ratio such that `threshold[k + 1] == (threshold[k] *
+		/// constant_ratio).max(threshold[k] + 1)` for all `k`.
+		///
```

Comment:
What properties does a geometric series have that inclines us to reccomend it? e.g. does it make it easier to reason about how often a nominator will move between bags?